cts/cts-scheduler.in

```diff
@@ -633,6 +633,7 @@ TESTS = [
         SchedulerTest("no_quorum_demote", "Promotable demotion and primitive stop with no-quorum-policy=\"demote\""),
         SchedulerTest("no-promote-on-unrunnable-guest", "Don't select bundle instance for promotion when container can't run"),
         SchedulerTest("leftover-pending-monitor", "Prevent a leftover pending monitor from causing unexpected stop of other instances"),
+        SchedulerTest("promotable-colocation-with-node-attribute", "Promote dependend clone on nodes belonging to a site that has a primary clone promoted"),
```

Comment:
Move this to the second patch, and I think you meant "Promote dependent" instead of "Promote dependend".